pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -0,0 +1,45 @@
+[id='con-pantheon-yaml-file_{context}']
+
+= The Pantheon configuration file
+
+In order to use a Git repository to import content to Pantheon, you need to create a configuration file in the root directory of your repository named `pantheon2.yml`. This yaml file indicates to Pantheon that it can interact with files in the repository.
+
+The configuration file enables the following:
+
+* Importing content from a Git repository into Pantheon
+* Uploading modules into Pantheon using the uploader tool
+
+The configuration file defines the following:
+
+`server`:: The URL of the Pantheon server.
+`repository`:: A unique ID to reference the URL of the repository. You specify the actual URL of the repository when you synchronize the repository the first time.
+`modules`:: Patterns for modules that will show up in the search results on Pantheon.
+`resources`:: Image files, legal information or other content that is not.
```

Comment:
Incomplete sentence!

Reply:
Good catch. I guess you don't like to play guessing games, huh?